README.md

```diff
@@ -185,16 +185,22 @@ This QR Code/link should then be embedded into your page using the Yoti widget t
 
 ```javascript
 
-yoticlient.getDynamicAttributeListQRCodeLink(dynamicAttributeListRequest)
-    .then((DynamicAttributeListResult) => {
-        const qrCode = DynamicAttributeListResult.getQRCodeLink();
-        const refId = DynamicAttributeListResult.getRefId();
-        const qr-code-div = document.getElementById('qr-code-div');
-        qr-code-div.innerHTML = qrCode;
-    })
+const dynamicPolicy = new Yoti.DynamicPolicyBuilder()
+  .withFullName()
+  .build();
 
-```
+const dynamicScenario = new Yoti.DynamicScenarioBuilder()
+  .withCallbackEndpoint('/profile')
```

Comment:
I know optional arguments aren't as clean in Node as other languages, but what do you think about making this parameter optional? And having the default if it's not set be `/profile`? I'm just thinking I can't envisage clients ever wanting to change that based on the current setup, so it's just adding an extra bit of config for them

Reply:
I think this path/URL would vary quite often between applications and might be more risky to set a default. We'd also need to document that the default is `/profile` and changing the default in future would be a breaking change as clients would start to depend on it. (I think it's safer to require that clients specify the path for their application)

Reply:
Ok sure, fair enough